### PR TITLE
GH Pages Styles update

### PR DESCRIPTION
**Pull request by Zejnilovic**
- Add style to code blocks
- Add other stylesheets for future - function for people who prefer dark screen or other 
- Add style to tables

### CODE REVIEW

File: _docs/1.0.0/usage-schema.md [hunk 1]
```diff
@@ -159,17 +155,13 @@ Data types
 
 The data type representing *String* values.
 
-| <sub><sup>Metadata keys:</sup></sub> | <sub><sup>[sourcecolumn](#sourcecolumn), [default](#default)</sup></sub> |
-|--------------------------------------|--------------------------------------------------------------------------|
-| | |
+<sup>**Metadata keys:** [sourcecolumn](#sourcecolumn), [default](#default)</sup>
```
Comment (benedeki) [inline _docs/1.0.0/usage-schema.md]: Why the `<sub>` removal? It places the small font text in the middle of the line.

Comment (Zejnilovic) [reply _docs/1.0.0/usage-schema.md]: This should be more attached to the text above it and big enough to easily read.

In all honesty, I don't like either `sup` or `sub` but did not come with a better solution. I was thinking of defining some special class for it using `<p>` tag or something similar and still thinking about it. This is a misuse.

**Comment by Zejnilovic** [comment]
> SonarCloud Quality Gate failed.
> 
> [Bug](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=BUG) [C](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=BUG) [1 Bug](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=BUG)
> [Vulnerability](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=VULNERABILITY) (and [Security Hotspot](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=SECURITY_HOTSPOT) to review)
> [Code Smell](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=CODE_SMELL) [0 Code Smells](https://sonarcloud.io/project/issues?id=AbsaOSS_enceladus&pullRequest=1204&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=AbsaOSS_enceladus&pullRequest=1204) No Coverage information
> [](https://sonarcloud.io/component_measures?id=AbsaOSS_enceladus&pullRequest=1204&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=AbsaOSS_enceladus&pullRequest=1204&metric=new_duplicated_lines_density&view=list)

I see my mistake. I just set the background twice. Will re repaired with the addition of custom footnote. It builds and shows everything correctly for now.

File: _docs/1.0.0/usage-schema.md (hunk 1 above)
Comment (benedeki) [reply _docs/1.0.0/usage-schema.md]: Markdown is purposefully simplistic. Which comes at a price of these limitations. And the line between creativity and misuse is thin ;-)
As there is no text on the same line anyway, not that important. :-)
(For the record `<sup><sub>` is not my invention, I found it somewhere on the net)

**Comment by Zejnilovic** [comment]
As @benedeki said. For using markdown, we are paying the price of not being able to have anything better than the `<sup>` currently there.  

Meaning, I don't plan to add any more "features" to this PR.

**Pull request merged**